.openzeppelin/unknown-421614.json

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
we can remove the arb goerli as it's not used

Reply:
I am curious to why the `arbitrum-sepolia.json` file is not present 🤔